### PR TITLE
[CoreBundle] Adding specs for InsufficientStockExceptionListener

### PR DESCRIPTION
**Pull request by mgonyan**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | enhancement
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | no
| License       | MIT
| Doc PR        | no

- Adding specs for InsufficientStockExceptionListener class
- Fixing CS

### CODE REVIEW

File: src/Sylius/Bundle/CoreBundle/EventListener/InsufficientStockExceptionListener.php [hunk 1]
```diff
@@ -23,24 +23,49 @@
  */
 class InsufficientStockExceptionListener
 {
+    /**
+     * @var UrlGeneratorInterface
+     */
     protected $router;
```
Comment (pjedrzejewski) [inline src/Sylius/Bundle/CoreBundle/EventListener/InsufficientStockExceptionListener.php]: Properties should be separated with blank lines.

File: src/Sylius/Bundle/CoreBundle/spec/EventListener/InsufficientStockExceptionListenerSpec.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+use Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * @author Manuel Gonzalez <mgonyan@gmail.com>
+ */
+class InsufficientStockExceptionListenerSpec extends ObjectBehavior
+{
+    private $redirectTo = 'url';
```
Comment (pjedrzejewski) [inline src/Sylius/Bundle/CoreBundle/spec/EventListener/InsufficientStockExceptionListenerSpec.php]: No need to use a property here.

**Comment by pjedrzejewski** [comment]
Please fix my comments. :+1: Good job! Thanks!

**Comment by mgonyan** [comment]
Comments has been fixed. Thanks :+1:

File: src/Sylius/Bundle/CoreBundle/EventListener/InsufficientStockExceptionListener.php (hunk 1 above)
Comment (mgonyan) [inline src/Sylius/Bundle/CoreBundle/EventListener/InsufficientStockExceptionListener.php]: :+1:

**Comment by pjedrzejewski** [comment]
Nice, I have restarted the failing builds.

**Comment by mgonyan** [comment]
@pjedrzejewski fixed

**Comment by pjedrzejewski** [comment]
Thank you Manuel! :+1:

**Pull request merged**